daemon/web/components/TerminalView/TerminalView.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './index.sass';
+
+const styles = {
+  textarea: {
```

Comment:
Let's follow camelcase naming convention here with `textArea`

Normally I might argue for this to be a generic class in a stylesheet, but I'll just note that having it here would allow easier themeing of the terminal view